llvm/lib/Transforms/IPO/RepoPruning.cpp

```diff
@@ -263,9 +254,71 @@ bool RepoPruning::runOnModule(Module &M) {
     }
   }
 
-  LLVM_DEBUG(dbgs() << "size of module: " << M.size() << '\n');
-  LLVM_DEBUG(dbgs() << "size of removed functions: " << NumFunctions << '\n');
-  LLVM_DEBUG(dbgs() << "size of removed variables: " << NumVariables << '\n');
+  LLVM_DEBUG(dbgs() << "Size of module: " << M.size() << '\n');
+  LLVM_DEBUG(dbgs() << "Number of removed functions: " << NumFunctions << '\n');
+  LLVM_DEBUG(dbgs() << "Number of removed variables: " << NumVariables << '\n');
+
+  return Changed;
+}
+
+static bool wasPruned(const GlobalObject &GO) {
+  if (const MDNode *const T = GO.getMetadata(LLVMContext::MD_repo_ticket)) {
+    if (const TicketNode *const MD = dyn_cast<TicketNode>(T)) {
```

Comment:
The types and variables names seem a little odd here. T (ticket?) is an MDNode, and MD (Metadata?) is a TicketNode! Is that right?

Reply:
Thanks, corrected these:
1) Rename `T` to `MD`;
2) Rename `MD` to `TN`;